modules/iam/main.tf

```diff
@@ -1,25 +1,46 @@
+locals {
+  role_name = format(
+    "%s-%s-%s",
+    var.project,
+    var.role_name == "policyname" ? var.policy : var.role_name,
+    var.environment
+  )
+  policy_name = format(
+    "%s-%s-%s",
+    var.project,
+    var.role_name == "policyname" ? var.policy : var.role_name,
+    var.environment
+  )
+  ddb_table_arn = format(
+    "arn:aws:dynamodb:%s:%s:table/%sVulnerableDomains%s",
+    data.aws_region.current.name,
+    data.aws_caller_identity.current.account_id,
+    replace(title(replace(var.project, "-", " ")), " ", ""),
+    title(var.environment),
+  )
+  ddb_ip_table_arn = format(
+    "arn:aws:dynamodb:%s:%s:table/%sIPs%s",
+    data.aws_region.current.name,
+    data.aws_caller_identity.current.account_id,
+    replace(title(replace(var.project, "-", " ")), " ", ""),
+    title(var.environment),
+  )
```

Comment:
Can these arns be collected via a data source instead of constructing the arn by hand?

Reply:
They don't exist, can we?

Reply:
Ok, I see what you say. Maybe we can have better ordering and if we create dynamo first then we can query.

Reply:
Will need dependency on modules and a little refactor. Right now the IPs table arn string is added independently of `var.ip_address`. If we use data sources, those will not exists.

Reply:
Moved to data sources.